src/main/java/edu/ucla/library/bucketeer/verticles/FinalizeJobVerticle.java

```diff
@@ -47,7 +47,9 @@ public void start() throws Exception {
 
                     // If we have someone waiting on this result, let them know
                     if (slackHandle.isPresent()) {
-                        final String iiifURL = myConfig.getString(Config.IIIF_URL);
+                        final String baseURL = myConfig.getString(Config.IIIF_URL);
+                        final String iiifURL = baseURL.substring(0, baseURL.lastIndexOf("iiif"));
```

Comment:
While we're currently using "iiif" as the IIIF service prefix, we can't assume that will always be there since we allow the IIIF URL to be configured (and others may not use the same prefix as we do (or we, even, may change in the future)). We could match on the first `/` after the `://` or we could put it into a URL and pull out the relevant parts.

Reply:
Modified code to use URL object to extract protocol and host with no assumptions about content of either.